website/docs/reference/resource-properties/tags.md

```diff
@@ -29,6 +25,7 @@ version: 2
 
 models:
   - name: model_name
+    # tags: [] # check out [these docs](resource-configs/tags) for more info
```

Comment:
FYI the markdown link rendered doesn't like this:
https://deploy-preview-173--docs-getdbt-com.netlify.app/reference/resource-properties/tags